tests/Smoke/CiIntegrationTest.php

```diff
@@ -157,15 +157,15 @@ public function testIntegration(
             preg_quote('Legend: ?-unknown, I-invalid file syntax (file ignored), S-skipped (cached or empty file), .-no changes, F-fixed, E-error', '/')
         );
 
-        static::assertRegExp($pattern, $result3->getError());
+        static::assertMatchesRegularExpression($pattern, $result3->getError());
```

Comment:
just to ensure - this change is done manually or automatically by one of the PHP CS Fixer rule ?

Reply:
Not it wasn't. It should have been, but it wasn't. 

There's no such `@PHPUnitXXMigration` rule-set that would do this, as far as I can tell.

Reply:
can you raise a feature request for it, please?

Reply:
Done: #5294